### DISCUSSION TITLE
Why can't you install all normal playonlinux programs?

### DISCUSSION THREAD

**Discussion by ghost** [PhoenicisOrg/phoenicis#2328] [Q&A]
In Phoenicis,there aren't things like Internet Explorer 1,why is that so?

**Comment by plata** [comment]
The way how apps are installed has been completely reworked. For example, it's now a lot easier to write an installer. As a result, PoL4 installers are not compatible.

**Reply by ghost** [reply]
ok

**Reply by Mannshoch** [reply]
May consider to find a way to migrate/convert old scripts into the new form?
